### PR TITLE
IOS-2922 Hide send feedback if not supported

### PR DESCRIPTION
**Pull request by tureck1y**

### CODE REVIEW

File: Tangem/Modules/Details/DetailsViewModel.swift [hunk 1]
```diff
@@ -188,9 +188,12 @@ extension DetailsViewModel {
 
     func setupSupportSectionModels() {
         supportSectionModels = [
-            DefaultRowViewModel(title: "details_chat".localized, action: openSupportChat),
-            DefaultRowViewModel(title: "details_row_title_send_feedback".localized, action: openMail),
+            DefaultRowViewModel(title: "details_chat".localized, action: openSupportChat)
```
Comment (megakoko) [inline Tangem/Modules/Details/DetailsViewModel.swift]: Missing trailing comma

File: Tangem/Modules/Details/DetailsViewModel.swift [hunk 2]
```diff
@@ -188,9 +188,12 @@ extension DetailsViewModel {
...
         ]
+
+        if cardModel.emailConfig != nil {
+            supportSectionModels.append(DefaultRowViewModel(title: "details_row_title_send_feedback".localized, action: openMail))
+        }
```
Comment (megakoko) [inline Tangem/Modules/Details/DetailsViewModel.swift:196]: NP: not for now but would it be better to have an optional openMail closure instead?

**Review by megakoko — DISMISSED** [review]

**Review by ba1ashov — DISMISSED** [review]

File: Tangem/Modules/Details/DetailsViewModel.swift (hunk 1 above)
Comment (tureck1y) [reply Tangem/Modules/Details/DetailsViewModel.swift]: ✅

**Review by megakoko — APPROVED** [review]

File: Tangem/Modules/Details/DetailsViewModel.swift (hunk 2 above)
Comment (tureck1y) [reply Tangem/Modules/Details/DetailsViewModel.swift:196]: We must hide this section entirely

Comment (megakoko) [reply Tangem/Modules/Details/DetailsViewModel.swift:196]: I mean controlling the visibility of this section based on whether the action itself is nil or not.

Comment (megakoko) [reply Tangem/Modules/Details/DetailsViewModel.swift:196]:
```
        if let openMail = self.openMail {
            supportSectionModels.append(DefaultRowViewModel(title: "details_row_title_send_feedback".localized, action: openMail))
        }
 
```

**Review by Andoran90 — APPROVED** [review]

File: Tangem/Modules/Details/DetailsViewModel.swift (hunk 2 above)
Comment (tureck1y) [reply Tangem/Modules/Details/DetailsViewModel.swift:196]: I think the optional action it's kind of the reverse of the logic. It's more straightforward to make a decision here based on the emailConfig and pass it to the action.

**Review by ba1ashov — APPROVED** [review]

**Pull request merged**